gemini/sample-apps/genwealth/deployment/enable-apis.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Enable Backend APIs
+PROJECT_ID=$(gcloud config get-value project 2> /dev/null)
+gcloud services enable iam.googleapis.com --project "${PROJECT_ID}"
```

Comment:
You might be able to put all of these in a single command.

Reply:
Looks like it will only accept up to 20 per batch, so I combined this into two batches.